.travis.yml

```diff
@@ -26,17 +26,5 @@ jobs:
         - npx aegir dep-check
```

Comment:
Can you remove the following line?

```js
npx aegir commitlint --travis
```

This is making the CI to fail. We have been discussing this, and we can remove that as this seems to be a difficulty to contributors